src/test/kotlin/com/r3/developers/csdetemplate/MyFirstFlowTest.kt

```diff
@@ -18,12 +18,12 @@ class MyFirstFlowTest {
         // Instantiate an instance of the Simulator
         val simulator = Simulator()
 
-        // Create Alice's and Bob HoldingIDs
+        // Create Alice's and Bob's HoldingIDs
         val aliceHoldingID = HoldingIdentity.Companion.create(aliceX500)
         val bobHoldingID = HoldingIdentity.Companion.create(bobX500)
 
-        // Create Alice and Bob's virtual nodes, including the Class's of the flows which will be registered on each node.
-        // We don't assign Bob's virtual node to a val because we don't need it for this particular test.
+        // Create Alice's and Bob's virtual nodes, including the Class's of the flows which will be registered on each node.
```

Comment:
Not Class's

- Does it need to be capitalized? (unsure myself)  
- Should it not be Classes, as use of ' indicates ownership, which doesn't look to be applicable in this sentence. including the Class of the flows....